api/controllers/SignupController.js

```diff
@@ -28,54 +31,58 @@ module.exports = {
     }
 
     // Upsert the user with the provided info
-    User.findOne({phone: phone})
+    User.findOne({ phone: phone })
       .then(function(result) {
         let user = {
           firstName: firstName,
-          phone: phone
+          phone: phone,
         };
 
         if (email) {
           user.email = email;
         }
 
-        if (referredBy) {
+        // The referredBy value can be updated if it has not yet been set, or if
+        // the user has not yet subscribed to Mobile Commons.
+        const canUpdateReferredBy =
```

Comment:
Yea. And the only time this route is currently called is from /join requests in aurora.